### ISSUE TITLE
Update the guidance on releases being optional on Record

### ISSUE THREAD

**Issue by yolile** [open-contracting/standard#1412]
In OCDS 1.2 the releases are now optional on Record in favor of allowing the publishers to publish only compile releases. We need to update the relevant guidance on this.

From https://github.com/open-contracting/standard/issues/1092#issuecomment-910907474:

>  it would be relevant to make the approach of only publishing compiled releases known on some of the change history or easy releases pages.

**Comment by yolile** [comment]
Should we add a caveat in the 'Easy releases' page as per https://github.com/open-contracting/standard/issues/1092#issuecomment-713248126 and add a new page to explain how and when to use a `compiledRelease`? (eg "Latest version only" (other names welcome)) That is because I think the 'Easy releases' page is already long and focuses on still publishing (but not storing) multiple releases per contracting process. If we add this new page, we will have:
- Change history (for the full versioning support)
- Easy releases (for partial versioning support)
- Latest version only (for not versioning at all)

**Comment by jpmckinney** [comment]
I never liked the name "easy releases", and we don't want even more terms. There are two main decisions to make:

1. Whether to have a change history (if not, link to new page about publishing compiled releases only)
1. Whether to have incremental updates (current "change history" page) or full updates (current "easy releases" page)

There should be a new "change history" section that discusses the two decisions, and why you would choose one or the other, and then link to the appropriate example.

This structure would make the decisions that publishers need to make much clearer. The easiness of implementation isn't the most important consideration, so I'd be happy to remove that word.

**Comment by yolile** [comment]
> and we don't want even more terms.

Agree, but I guess we still need a title for the new/replaced sections, rigth?  Something like:

The new section with the approaches discussion -> "Change history"
The current "Change history" -> "Incremental updates"
"Easy releases" -> "Full updates" 
The new section with the compiled release only example -> ?

Or what do you think?

**Comment by jpmckinney** [comment]
Yes, and the new page can be "No change history" ?

**Comment by yolile** [comment]
I've drafted [here](https://docs.google.com/document/d/1h60L7iheIiz6nXKco6NQ85UzfCZJnWzxFYRztEzd0BQ/edit) the content for the new two pages (The new section with the approaches discussion -> "Change history" and The new section with the compiled release only example -> "No change history"). @duncandewhurst maybe you can take a look at it first?

**Comment by duncandewhurst** [comment]
@yolile I've added some comments and suggestions - happy to discuss!

**Comment by yolile** [comment]
Should we change https://standard.open-contracting.org/latest/en/schema/merging/#merge-routine as well? At least the first sentence that says "To create a compiled or versioned release, you **must**:"

**Comment by jpmckinney** [comment]
I think we'll need to word a bit carefully, because if individual releases are optional, then compiled releases can be created directly. (Versioned releases, on the other hand, don't make any sense without individual releases.)

**Comment by yolile** [comment]
Related: https://github.com/open-contracting/standard/issues/1364

**Comment by duncandewhurst** [comment]
Just noting that the outstanding task here is to rename some pages and update their introductory paragraphs per https://github.com/open-contracting/standard/pull/1531#pullrequestreview-1079418478

**Comment by jpmckinney** [comment]
I think the main outstanding task is finishing #1531.
